bigstream/transform.py

```diff
@@ -243,7 +243,7 @@ def compose_displacement_vector_fields(
     second_field,
     first_spacing,
     second_spacing,
-    extrapolate_with_nn=True
+    **kwargs,
 ):
```

Comment:
there was a check before that if extrapolate_with_nn was missing from kwargs you set it to True - that was the hack I didn't like when I explicitly passed it. Do you still need that

Reply:
Yeah, the hack is still there. The idea is that the user should be able to control anything they want about how `apply_transform` is called, but the default for `extrapolate_with_nn` at least should be True. We need to keep `kwargs` so the user has control over `apply_transform`. We could keep `kwargs` and just make that particular argument explicit. Want me to make that change?